docs/serverless/08-07-sync-function-with-gitops.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Sync Git resources with the cluster using a GitOps operator
+type: Tutorials
+---
+
+This tutorial shows how you can automate the deployment of local Kyma resources on a cluster using the GitOps logic. You will use [Kyma CLI](https://kyma-project.io/docs/cli/) to create an inline Python Function with a trigger. You will later push both resources to a GitHub repository of your choice and set up a GitOps operator to monitor the given repository folder and synchronize any changes in it with your cluster. For the purpose of this tutorial, you will install and use the [Flux](https://docs.fluxcd.io/en/1.17.1/tutorials/get-started.html) GitOps operator and a lightweight [k3d](https://k3d.io/) cluster.
+
+> **TIP:** Although this tutorial uses Flux to synchronize Git resources with the cluster, you can use an alternative GitOps operator for this purpose, such as [Argo](https://argoproj.github.io/argo-cd/).
+
+## Prerequisites
+
+All you need before you start is to have the following:
+
+- CLI
+- brew
+- Git repository
+
+## Steps
+
+These sections will lead you through the whole installation, configuration, and synchronization process. You will first install k3d and create a cluster for your custom resources (CRs). Then, you will need to apply the necessary Custom Resource Definitions (CRDs) from Kyma to be able to create Functions and triggers. Finally, you will install Flux and authorize it with the `write` access to your GitHub repository in which you store the resource files. Flux will automatically synchronize any new changes pushed to your repository with your k3d cluster.
+
+### Install and configure a k3d cluster
+
+1. Install k3d using brew:
+
+  ```bash
+  brew install k3d
+  ```
+
+2. Cluster a default k3d cluster with a single server node:
+
+  ```bash
+  k3d cluster create {CLUSTER-NAME}
+  ```
+
+This command also sets your context to the newly created cluster. Run this command to display the cluster information:
+
+  ```bash
+  kubectl cluster-info
+  ```
+
+3. Apply the Kyma's `functions.serverless.kyma-project.io` and `triggers.eventing.knative.dev` CRDs from the sources in the [`kyma`](https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials/files) repository. You will need them to create the Function and Trigger CRs on the cluster.
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/master/resources/cluster-essentials/files/functions.serverless.crd.yaml && kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/master/resources/cluster-essentials/files/triggers.eventing.knative.dev.crd.yaml
+  ```
+4. Run this command to make sure the CRs were applied:
```

Comment:
```suggestion
4. Run this command to make sure the CRs are applied:
```

Reply:
I'm not sure about this one tbh. Shouldn't we use the present tense whenever it's possible? 🤔 
(applies to the whole document)

Reply:
It is difficult to avoid future tense everywhere, though.